lib/algolia.ex

```diff
@@ -41,8 +41,9 @@ defmodule Algolia do
 
   defp host(:read, 0), do: "#{application_id()}-dsn.algolia.net"
   defp host(:write, 0), do: "#{application_id()}.algolia.net"
+  defp host(:insights, 0), do: "insights.algolia.io"
```

Comment:
Can you double check if the retry URL is supposed to be `"#{application_id()}-#{curr_retry}.algolianet.com"`? Currently this will only match the first request so retries will go to algolianet.com. Googling algolianet.com it seems this library is implemented the recommended strategy in https://www.algolia.com/doc/rest-api/search/#hosts however I don't know if it applies to insight api requests (since that is also a newer feature). Modifying the test indicates not.

Reply:
yeah, seems like it doesn't apply to insights. retries should still go to `insights.algolia.io`

Reply:
changed and tested by changing the retry value in `push_events`.